api/proto/config/deck.proto

```diff
@@ -44,6 +44,9 @@ message Deck {
   // Configuration for UI-related feature flags.
   Features feature = 10;
 
+  // Configuration for ManagedDelivery
```

Comment:
I think this should be "managed delivery" in the comments here (and in a few places below).

I'm only being particular about the comments because they generate user-facing documentation.